packages/wonder-blocks-clickable/src/components/clickable.tsx

```diff
@@ -156,65 +155,67 @@ const StyledLink = addStyle(Link);
  * </Clickable>
  * ```
  */
-export default class Clickable extends React.Component<Props> {
-    static defaultProps: DefaultProps = {
-        light: false,
-        disabled: false,
-    };
 
-    getCorrectTag: (
+const Clickable = React.forwardRef(function Clickable(
+    props: Props,
+    ref: React.ForwardedRef<any>,
```

Comment:
Note: I used `any` type because the inner element has 3 possible types. I'm not sure if this is the best way to accomplish this or if I should try to limit it and typecast based on the situation.